vnext/ReactWindows-UWP.sln

```diff
@@ -96,81 +98,97 @@ Global
 	EndGlobalSection
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
 		Debug|ARM = Debug|ARM
+		Debug|ARM64 = Debug|ARM64
```

Comment:
>Debug|ARM64 = Debug|ARM64 [](start = 2, length = 25)

Please sync to latest, which has this change, which turns off RTTI:
[https://github.com/microsoft/react-native-windows/commit/461a76c1f7087ff574fe66a298913509383fa981](https://github.com/microsoft/react-native-windows/commit/461a76c1f7087ff574fe66a298913509383fa981)
You'll need to also turn off RTTI for the new ARM64 build flavor.